wiki/Game_mode/de.md

```diff
@@ -53,6 +50,12 @@ In *osu!catch* fallen Früchte verschiedener Sorten vom oberen Bildschirmrand he
 
 ![Gameplay von osu!mania](/wiki/shared/mania-gameplay.jpg "osu!mania-Gameplay")
 
+## Andere Spielmodi
```

Comment:
I think the accepted translation for custom in this context is `Benutzerdefiniert`